Dockerfile

```diff
@@ -4,8 +4,10 @@ ENV PORT=8080
 
 ARG ENVIRONMENT
 ENV ENVIRONMENT=$ENVIRONMENT
+ENV SENTRY_ENVIRONMENT=$ENVIRONMENT
 RUN echo Environment: $ENVIRONMENT
 
 ARG VERSION
 ENV VERSION=$VERSION
+ENV SENTRY_RELEASE=$VERSION
```

Comment:
looks like you set it, but you don't use it.
we only read it automatically on the sentry_flutter package, this would be ideally moved to the `sentry` package, create an issue please :)

Reply:
Yeah I thought Sentry would pick it up automatically. Will raise the issue. thanks.

Reply:
https://github.com/getsentry/sentry-dart/issues/306